source/jormungandr/tests/equipment_tests.py

```diff
@@ -315,3 +317,55 @@ def test_equipment_reports_with_forbidden_uris(self):
         for equipment_report in equipment_reports:
             is_valid_equipment_report(equipment_report)
         self._check_equipment_report(equipment_reports, expected_result)
+
+    def test_equipment_reports_with_uri(self):
+        """
+        with uri filter
+        """
+
+        mock_equipment_providers(
+            equipment_provider_manager=self.equipment_provider_manager("main_routing_test"),
+            data=standard_mock_elevator_data,
+            code_types_list=["TCL_ASCENSEUR", "TCL_ESCALIER"],
+        )
+        response = self.query_region(default_line_uri_filter + 'equipment_reports?' + default_date_filter)
+
+        # Expected response
+        stopA_equipment_details = [
+            ("5", "elevator", "unknown"),
+            ("1", "escalator", "unknown"),
+            ("2", "escalator", "unknown"),
+            ("3", "escalator", "unknown"),
+            ("4", "escalator", "unknown"),
+        ]
+        stopb_equipment_details = [
```

Comment:
Super-Minor:
```suggestion
        stopB_equipment_details = [
```

Reply:
all tests are `stopb_equipment_details` name so I keep it